Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -464,50 +566,60 @@ + (NSString*)_getStringFromClipboard {
     return [stringData autorelease];
 }
 
-+ (void)_setStringToClipboard:(NSString*)stringData {
++ (NSString*)_getStringFromClipboard {
+    WADDataPackageView* wasDataPackageView = _getClipboardContent();
+    return [UIPasteboard _getStringFromDataPackageView:wasDataPackageView];
+}
+
++ (void)_setStringOnDataPackage:(NSString*)stringData DataPackage:(WADDataPackage*)dataPackage {
```

Comment:
nit: actually, shouldn't this be: _setString:(NSString*)stringData onDataPackage:(WADDataPackage*)dataPackage

---
In reply to: [87644537](https://github.com/Microsoft/WinObjC/pull/1358#discussion_r87644537) [](ancestors = 87644537)